.github/workflows/windows.yaml

```diff
@@ -9,15 +9,17 @@ jobs:
       DOWNLOAD_CACHE: 'd:\downloadcache'
     runs-on: windows-latest
     steps:
+      - shell: pwsh
+        run: |
+          echo "GOPATH=D:/a/_temp/go/work" >> $env:GITHUB_ENV
+          echo "GOCACHE=D:/a/_temp/go/cache" >> $env:GITHUB_ENV
```

Comment:
nit: Would it be possible to use the `env` config key for this step to define these env vars instead?

Reply:
Good call, I tunnel-visioned to hard here.